ShapesProject/src/test/java/ShapeTest.java

```diff
@@ -0,0 +1,11 @@
+public class ShapeTest {
```

Comment:
This is not a test! Tests run with junit and have test methods not main methods!